### PR TITLE
Revert "Sliding window for full Calo system"

### PR DESCRIPTION
**Pull request by clementhelsens**
Reverts FCC-hh-framework/FCCSimJobs#30
Need to check the SW generalisation better

### CODE REVIEW

**Comment by clementhelsens** [comment]
@CoralieNeubueser, I revert the PR as I suspect there is something wrong in the way the generalisation of the sliding window is done. As it affects the E-Cal, and as you have not shown (at least I haven't seen) that the results stays identical, I'd like @zaborowska to help in the review.

File: config/recSlidingWindow.py [hunk 1]
```diff
@@ -192,119 +142,82 @@
...
-                                          inhits = "HCalBarrelPositionsNoise",
-                                          outhits = "newHCalBarrelCellsNoise")
-    
-   #Create calo clusters
+    #Create calo clusters
     from Configurables import CreateCaloClustersSlidingWindow, CaloTowerTool
     from GaudiKernel.PhysicalConstants import pi
     towersNoise = CaloTowerTool("towersNoise",
-                           deltaEtaTower = 0.01, deltaPhiTower = 2*pi/256.,
+                           deltaEtaTower = 0.01, deltaPhiTower = 2*pi/704.,
```
[the remark this answers is not in the document]
Comment (CoralieNeubueser) [reply config/recSlidingWindow.py]: True, let me change it back, such that we have it consistent with the already reconstructed samples.

File: config/recSlidingWindow.py [hunk 2]
```diff
@@ -192,119 +142,82 @@
...
-    towersNoise.hcalBarrelCells.Path = "newHCalBarrelCellsNoise"
+    # towersNoise.hcalBarrelCells.Path = "HCalBarrelCells"
+    # towersNoise.hcalExtBarrelCells.Path = "HCalExtBarrelCells"
+    towersNoise.hcalBarrelCells.Path = "emptyCaloCells"
     towersNoise.hcalExtBarrelCells.Path = "emptyCaloCells"
     towersNoise.hcalEndcapCells.Path = "HCalEndcapCells"
     towersNoise.hcalFwdCells.Path = "HCalFwdCells"
     createclustersNoise = CreateCaloClustersSlidingWindow("CreateCaloClustersNoise",
-                                                          towerTool = towersNoise,
-                                                          nEtaWindow = winEta, nPhiWindow = winPhi,
```
[the remark this answers is not in the document]
Comment (CoralieNeubueser) [reply config/recSlidingWindow.py]: Sure, but I set the sizes following your default sizes where the seeding and final was pretty much the same.

File: config/recSlidingWindow.py [hunk 3]
```diff
@@ -64,22 +43,20 @@
 detectors_to_use=[path_to_detector+'/Detector/DetFCChhBaseline1/compact/FCChh_DectEmptyMaster.xml',
                   path_to_detector+'/Detector/DetFCChhTrackerTkLayout/compact/Tracker.xml',
                   path_to_detector+'/Detector/DetFCChhECalInclined/compact/FCChh_ECalBarrel_withCryostat.xml',
-                  path_to_detector+'/Detector/DetFCChhHCalTile/compact/FCChh_HCalBarrel_TileCal.xml']
-if not noise:
-    detectors_to_use += [path_to_detector+'/Detector/DetFCChhHCalTile/compact/FCChh_HCalExtendedBarrel_TileCal.xml']
```
[the remark this answers is not in the document]
Comment (CoralieNeubueser) [reply config/recSlidingWindow.py]: When running the NestedVolumeTool to get all cells and add noise in HCal Barrel, there is a confusion of volumes when the EB is build too.. its an issue I have to fix, but for now the EB is not to build when noise is suppose to be added in B.

File: config/recSlidingWindow.py (hunk 2 above)
[the remark this answers is not in the document]
Comment (CoralieNeubueser) [reply config/recSlidingWindow.py]: Sure, but I set the sizes following your default sizes where the seeding and final was pretty much the same.

**Comment by CoralieNeubueser** [comment]
I think we can close this pr, and I open another one with the change of the window granularity.

**Comment by clementhelsens** [comment]
I merged, but we need a proper implementation asap!

**Pull request merged**